### PR TITLE
Expand footer

### LINKED ISSUE #5705: Footer: Expand area below last footer line
To make scrolling down and accessing the language switcher more comfortable, please expand the area below the last text and button line.

Try out maybe 30-40 px and see how it looks and feels.

Screenshot from 2020-11-21 06-33-08

### PR DESCRIPTION
**Pull request by daretobedifferent18**
Fixes #5705 

#### ScreenShot

Screenshot from 2020-11-22 07-53-36

### CODE REVIEW

File: app/templates/components/footer-main.hbs [hunk 1]
```diff
@@ -40,6 +40,25 @@
           </a>
         </div>
       </div>
+    </div>
+  </div>
+  <div class="ui inverted section divider" style="margin-bottom: 55px!important;"></div>
```
Comment (daretobedifferent18) [inline app/templates/components/footer-main.hbs:45]: Changed this

Comment (daretobedifferent18) [reply app/templates/components/footer-main.hbs:45]: @iamareebjamal Pls see the changes

Comment (daretobedifferent18) [reply app/templates/components/footer-main.hbs:45]: @iamareebjamal I have added 55px which looks good to me

File: app/templates/components/footer-main.hbs [hunk 2]
```diff
@@ -40,6 +40,26 @@
...
+          {{t 'Contact'}}
+          {{t 'Refund Policy'}}
+          {{t 'Privacy'}}
+        </span>
+        {{#each this.footerPages as |page|}}
+          <LinkTo @route="pages" @model={{page.url}} class="item">
+            {{t-var page.name}}
+          </LinkTo>
+        {{/each}}
+      </div>
```
Comment (iamareebjamal) [inline app/templates/components/footer-main.hbs:62]: Why this much code added?

Comment (daretobedifferent18) [reply app/templates/components/footer-main.hbs:62]: i Don't there how was a merge conflict and it happened

Comment (daretobedifferent18) [reply app/templates/components/footer-main.hbs:62]: I only added style one

**Comment by daretobedifferent18** [comment]
i think PR is already there for this and here lot of complexity is formed so closing this

**Pull request closed without merging**